style.css

```diff
@@ -1,225 +1,658 @@
+}
+
+a.logo {
+  display: flex;
+  font-family: "Poppins", sans-serif;
   font-style: normal;
-  color: #344563;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  align-self: center;
+  color: #6070ff;
+  text-decoration: none;
+  margin-left: 24px;
+}
+
+.burger {
+  display: block;
+  color: #6070ff;
+  padding-top: 5px;
+  cursor: pointer;
+  position: absolute;
+  z-index: 1000;
+  right: 10%;
+  font-size: 2rem;
+}
+
+.navbar .fa-bars {
+  color: var(--primary);
+  font-size: 16px;
+}
+
+.menu {
+  display: block;
+  list-style: none;
+  position: fixed;
+  padding: 4rem;
+  top: -120%;
+  left: 0;
+  background: #6070ff;
+  width: 100%;
+  height: 100%;
+  opacity: 0.9;
+}
+
+nav a {
+  margin-bottom: 40px;
+  font-family: "Poppins", sans-serif;
+  font-size: 32px;
+  font-style: normal;
+  font-weight: 600;
+  line-height: 44px;
+  letter-spacing: 0;
+  text-align: left;
+  color: white;
+  text-decoration: none;
+  display: block;
+}
+
+.navigation nav {
+  position: fixed;
+  top: -120%;
+  width: 100%;
+  height: 100%;
+  background-color: #6070ff;
+  opacity: 0.9;
+}
+
+.fa-times::before,
+.fa-xmark::before {
+  content: "\f00d";
+  color: #fff;
+}
+
+.navigation nav.nav-toggle {
+  top: 0;
+}
+
+.desktop-menu {
+  display: none;
+}
+
+.main {
+  background: #f7f7f9;
+  padding: 0;
+  margin: 0;
+}
+
+.headline {
+  height: 604px;
+  background-image: url("images/header-shapes\ mobile.png");
+  background-color: #fff;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  background-repeat: no-repeat;
+  border: 1px solid #f7f7f9;
+  border-bottom-left-radius: 25% 50px;
+  background-size: 100%;
 }
 
-h1 {
-  font-family: poppins, sans-serif;
+.primary1 {
+  display: flex;
+  font-family: "Poppins", sans-serif;
   font-style: normal;
   font-weight: 700;
   font-size: 40px;
   line-height: 52px;
-  display: flex;
+  width: 327px;
+  height: 150px;
   align-items: center;
   color: #172b4d;
-  position: relative;
-  bottom: -35px;
-  margin-top: -4px;
+  margin: 114px 24px 12px 24px;
 }
 
-p#one {
+.headline p {
   font-family: "Poppins", sans-serif;
   font-style: normal;
   font-weight: 400;
   font-size: 16px;
   line-height: 24px;
   color: #344563;
-  position: relative;
-  bottom: -15px;
+  margin: 12px 24px 12px 24px;
+}
+
+.social-media {
+  display: flex;
+  flex-direction: column;
+  margin: 12px 24px 12px 24px;
 }
 
-p#two {
+.social-media p {
   font-family: "Poppins", sans-serif;
   font-style: normal;
   font-weight: 500;
   font-size: 16px;
   line-height: 24px;
   color: #7f8cff;
   margin-left: 0;
-  position: relative;
-  bottom: -20px;
 }
 
-section.Headline {
+.social-media-icons {
+  display: flex;
+  margin-top: 12px;
+  justify-content: space-between;
+  align-items: baseline;
+  list-style-type: none;
+  gap: 10px;
+}
+
+.works {
+  display: grid;
+  background: #f7f7f9;
+  padding-top: 75px;
+  grid-column-gap: 25px;
+  justify-content: center;
+}
+
+.card-works {
```

Comment:
[OPTIONAL] The width of the work section looks smaller compared to that of the figma design as shown below. Kindly updat the width to match the original design :wink: 

 Your design            |  The design template
:-------------------------:|:-------------------------:
![Screenshot from 2022-11-05 21-50-28](https://user-images.githubusercontent.com/65233860/200142903-d14d016d-1f48-4629-a573-a12055578538.png)  |  ![Screenshot from 2022-11-05 21-52-12](https://user-images.githubusercontent.com/65233860/200142907-044c197e-0546-4486-a920-8e913e44be1d.png)


 Your design            |  The design template
:-------------------------:|:-------------------------:
![Screenshot from 2022-11-05 21-50-55](https://user-images.githubusercontent.com/65233860/200142922-de2ddda4-8723-41b0-9318-0c567637bcc9.png)  |  ![Screenshot from 2022-11-05 21-52-27](https://user-images.githubusercontent.com/65233860/200142930-e664e8a3-259c-4202-a8a8-50069209649e.png)